src/api/type-defs/inputs/CreateLabelsInput.ts

```diff
@@ -13,8 +13,9 @@ export default /* GraphQL */ `
     labeledDate: Date
     validation: ValidationInput
     userId: ID
-    imageId: ID
+    imageId: ID!
```

Comment:
Required to make this work correctly:

https://github.com/tnc-ca-geo/animl-api/blob/76df33d98667de6bfdf837bb55fec9bbbf0226d9/src/api/db/models/Image.ts#L689-L702